model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_facade.py

```diff
@@ -82,6 +83,11 @@ def search_bit_width(graph_to_search_cfg: Graph,
 
     # Set graph for MP search
     graph = copy.deepcopy(graph_to_search_cfg)  # Copy graph before searching
+    # TODO: The handle of mixed precision with the fused graph will be in a separate PR. Currently, the bit-width
```

Comment:
also needs to be integrated with the BOPs virtual graph

Reply:
If I understand correctly, the integration with the BOPs, is simply to build the virtual graph from the fused graph.